app/src/main/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentPresenter.kt

```diff
@@ -236,7 +236,11 @@ class TopicLessonsFragmentPresenter @Inject constructor(
           ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES -> ChapterViewType.CHAPTER_LOCKED
           ChapterPlayState.COMPLETED -> ChapterViewType.CHAPTER_COMPLETED
           ChapterPlayState.IN_PROGRESS_SAVED -> ChapterViewType.CHAPTER_IN_PROGRESS
-          else -> ChapterViewType.CHAPTER_NOT_STARTED
+          ChapterPlayState.IN_PROGRESS_NOT_SAVED -> ChapterViewType.CHAPTER_IN_PROGRESS
```

Comment:
You can combine these together, e.g.:

```kotlin
ChapterPlayState.IN_PROGRESS_SAVED, ChapterPlayStage.IN_PROGRESS_NOT_SAVED ->
  ChapterViewType.CHAPTER_IN_PROGRESS
```

Ditto for the others (where relevant).

Reply:
done